synapse/pipeline.yml

```diff
@@ -527,3 +527,31 @@ steps:
 #
 #  - label: Trigger webhook
 #    command: "curl -k https://coveralls.io/webhook?repo_token=$COVERALLS_REPO_TOKEN -d \"payload[build_num]=$BUILDKITE_BUILD_NUMBER&payload[status]=done\""
+
+  ################################################################################
+  #
+  # Complement Test Suite
+  #
+  ################################################################################
+
+  - command:
+      - "bash scripts-dev/complement.sh"
```

Comment:
Careful, as this allows nasty Synapse PRs to execute arbitrary stuff.

Reply:
Ah, yes classic. I've just move the commands into the pipeline then.